### ISSUE TITLE
GT-means 测试问题

### ISSUE THREAD

**Issue by RenzhiHu111** [XunpengYi/Diff-Retinex#3]
您好，我想问一下有哪几篇低光增强算法是用了下面GT-means的测试方法
 # some method get very high PSNR (more than 27), check this in the code
        if args.use_gtmeans is True:
            normal_I = val_data['high'].detach().cpu().numpy().squeeze(0)*255.0
            normal_I_means = np.mean(normal_I)
            pred_I_means = np.mean(I)
            I = I * normal_I_means / pred_I_means

**Comment by XunpengYi** [comment]
您可以参照对应算法的实现代码。例如，KinD和LLFlow都实现了相应的接口。
